src/drive/web/modules/views/hooks.js

```diff
@@ -49,11 +49,8 @@ export const useFileWithPath = fileId => {
   const parentResult = useQuery(parentQuery.definition, parentQuery.options)
   const parentData = parentResult.data
 
-  if (isQueryLoading(fileResult) || isQueryLoading(parentResult)) {
-    return { data: null }
-  }
-
   return {
+    ...fileResult,
```

Comment:
I don't understand why returning `{data: null}` will trigger re-render but not `data: { undefined}`? Can you help me?

Reply:
Franchement je sais pas trop, j'ai assez galéré à comprendre et ayant passé pas mal de temps, voilà où j'ai abouti. Peut-être qu'une amélioration serait de faire le hook plutôt de cette manière https://github.com/cozy/cozy-drive/blob/master/src/drive/web/modules/views/OnlyOffice/useConfig.js

J'ai noté deux choses :
- la requête useQuery est ré-exécuté avec son lastupdate qui change, alors qu'on devrait récupérer le résultat dans le store non ? Comme elle est ré-exécuté, le composant se re-render.
- quand on retourne un object qu'on reconstruit à chaque fois comme `{data: {...result}}` par exemple, j'ai l'impression que react considère que c'est un nouvel object et non le même.

Comme fonctionnellement on a l'attendu et qu'on a une deadline un peu serré je serai plutôt d'avis de revenir là-dessus éventuellement à la fin.